sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
+
+    game_seconds_per_second = 1.4
+    if replay.expansion == 'LotV':
+        game_seconds_per_second = 1.4
```

Comment:
I just can't stand magic constants in programming, but maybe this is where I learn their usefulness. 😉 

Meanwhile, I actually managed to produce avg apm equal to in-game from a recent replay, so now I just need the app to understand.